### ISSUE TITLE
Adapt Switch model to output a default value

### ISSUE THREAD

**Issue by jlbouchot** [dynawo/dynawo#2925]
As part of the voltage control/optimisation process, a switch , controlling the monitored voltage, is used to adapt monitoring to a given state or a predicted state. 

Within the context of handling ACMCs, one needs to rethink the switch in its current form into something which will trigger an action from an ACMC (self / capa) based on both a predicted value (what if we were to increase the current tension plan) which is looked at only if no surtension is noticed. 

Basically, instead of switching from u1 to u2 based on 2 booleans (= the current state) we will output a default value within a parametrised deadband if we know no action is possible in the capa/self agent.
